portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
```

Comment:
nit: you might want a different title for each page. Using the same title in all pages is less helpful

Reply:
Definitely.